Tools/sitl_run.sh

```diff
@@ -6,8 +6,9 @@ sitl_bin="$1"
 debugger="$2"
 program="$3"
 model="$4"
-src_path="$5"
-build_path="$6"
+world="$5"
+src_path="$6"
+build_path="$7"
```

Comment:
For backwards compatibility you could add it as the last argument and assume a default if it is not provided.